zerver/lib/export.py

```diff
@@ -1440,7 +1440,7 @@ def export_uploads_and_avatars(
         if user is None:
             export_realm_icons(
                 realm,
-                local_dir=os.path.join(settings.LOCAL_UPLOADS_DIR),
+                local_dir=os.path.join(settings.LOCAL_UPLOADS_DIR, "avatars"),
```

Comment:
I don't totally understand how this line relates to the rest of this commit.

Reply:
Because of this:
```python3
def export_realm_icons(realm: Realm, local_dir: Path, output_dir: Path) -> None:
    records = []
    dir_relative_path = zerver.lib.upload.upload_backend.realm_avatar_and_logo_path(realm)
    icons_wildcard = os.path.join(local_dir, dir_relative_path, "*")
```

`zerver.lib.upload.upload_backend.realm_avatar_and_logo_path` no longer includes `avatars`, so we stitch it onto `local_dir` instead.  I've added a comment to the commit message.